CIP-0010/CIP-0010.md

```diff
@@ -0,0 +1,61 @@
+---
+CIP: 10
```

Comment:
This shouldn't show up as a new file in the history so this PR needs to be updated

Reply:
This should be sorted now, thanks for flagging any other issues as needed.